src/pages/Login/AuthButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { useAuthContext } from "../../context/AuthContext";
+
+const AuthButton = () => {
+  const history = useHistory();
+  const { user, signout } = useAuthContext();
+
+  return user.isAuthenticated ? (
+    <p>
+      Welcome!{" "}
+      <button
+        type="button"
+        onClick={() => {
+          signout(() => history.push("/"));
+        }}
+      >
+        Sign out
+      </button>
+    </p>
```

Comment:
♻️ Could you replace this with a Material-UI button? Also, the signout button.
https://material-ui.com/components/buttons/#button

Reply:
Buttons replaced with Material UI

Reply:
🔧 Please link the commit. Also, I don't see the change.

Reply:
commit:  f45b314c2394685c40b9cac7c99d82ea5852972c